### DISCUSSION TITLE
(gtfs, gtfs-rt, gbfs, mds) is not a valid FeedSpecTypes

### DISCUSSION THREAD

**Discussion by danthonywalker** [transitland/transitland#414] [Q&A] [answered]
Hello,

I am trying to get up and running with Transitland's API. When I try to fetch from `https://transit.land/api/v2/rest/feeds.json?spec=gtfs` I get a **500** status code with the following response: `{"error":"gtfs is not a valid FeedSpecTypes"}`. This is the case for all spec enum values.

The documentation for feeds clearly states that this should be possible, and the 500 indicates some type of server-side problem. Is this some type of error on my end?

**Comment by drewda** [comment]
Thanks for reporting this problem.

For the time being, please use all-caps for the spec value. For example: `?spec=GTFS`

We are looking into this and will provide another update.

**Reply by drewda** [reply, marked as answer]
We have revised the relevant code so that both `?spec=GTFS` and `?spec=gtfs` should now work.

(Users of the Transitland v2 GraphQL API will recognize that the all-caps version of GTFS is required because the GraphQL API is strongly typed. Behind the scenes, the REST API uses the GraphQL API, and so it was exposing this requirement. We have changed this so that the REST API is case insensitive while the GraphQL API will remain case sensitive.)

**Comment by danthonywalker** [comment]
Thank you. The work that you people do is very awesome.
